images/golang-legacy/Dockerfile

```
@@ -1,4 +1,5 @@
-FROM bootstrap-legacy
+ARG ARCH
```

nit: I think a default would make sense.
```suggestion
ARG ARCH=amd64
```